include/boost/graph/iteration_macros.hpp

```diff
@@ -17,45 +17,6 @@
 #define BGL_FIRST(linenum) (BGL_RANGE(linenum).first)
 #define BGL_LAST(linenum) (BGL_RANGE(linenum).second)
 
-/*
-  BGL_FORALL_VERTICES_T(v, g, graph_t)  // This is on line 9
```

Comment:
Is there a reason for deleting this comment block when it's a change not part of any of the PR's?